docker-compose/monitor/docker-compose-v2.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    # TODO: Need to use the local image here while making changes 
+    image: jaegertracing/jaeger
+    volumes:
+      - "./jaeger-ui.json:/etc/jaeger/jaeger-ui.json" # Do we need this for v2 ? Seems to be running without this.
+      - "./spm-v2.yml:/etc/jaeger/spm-v2.yml"
+    command: ["--config", "/etc/jaeger/spm-v2.yml"]
+    environment:
+      - METRICS_STORAGE_TYPE=prometheus
+      - PROMETHEUS_SERVER_URL=http://prometheus:9090
+      - PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR=${PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR:-true}
```

Comment:
is this still needed?

Reply:
Nope. I think this is enabled by default now.